frontend/src/app/services/event-list.ts

```diff
@@ -60,8 +80,33 @@ export function updateEventData(eventInput: EventInput) {
   } else {
     eventData.tooManyDeathsDays = 0;
   }
+
+  eventData.showAntivaxEvent = eventInput.date > '2020-01-10'
+    && isEventTriggerActive(eventInput, ANTIVAX_WITH_CAMPAIGN_TRIGGER)
+    && isEventTriggerActive(eventInput, ANTIVAX_WITHOUT_CAMPAIGN_TRIGGER)
+    && probability(0.02);
 }
 
+const antivaxEvents = [
+  {
+    title: 'Toto jsou fakta: vakcína proti koronaviru je tvořena fragmenty tkání z potracených lidských plodů!',
+    text: 'Ve společnosti se šíří hoax o tom, že vakcína obsahuje látky z nenarozených dětí.',
+    help: 'Rychlost vakcinace se snižuje.',
+    mitigations: [
+      {label: 'Ok', vaccinationPerDay: -0.0002, timeout: 1},
```

Comment:
Is it right the effect of `vaccinationPerDay` is permanent, whereas other `MitigationEffect` keys are only temporary while the event mitigation is active?

If so, it should be made reworked (see https://github.com/swehq/corona-game/blob/master/frontend/src/app/services/simulation.ts#L179) - unified. Btw, we have `oneTimeEffect`.

Reply:
attn @daeglin